File: Level074/home/task2.js
`Given a non-empty array of integers, return the result of multiplying the values together in order. Example:

[1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24`


// my code:

function grow(arr){

  res = 1;

  for (i = 0; i < arr.length; i++) {
    
   res *= arr[i];
    
  }
  
  return res;
}

File: Level36/class/task2.py
"""Given a non-empty array of integers, return the result of multiplying the values together in order. Example:

[1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24
"""



# solution:

# def grow(arr):
#     pass



# My code:

def grow(x):
    sum = 1
    
    for i in x:
        sum *= i
    return sum

#grow([1,2,3,4])